In [20]:
import json

lines = []

with open("train.jsonl", mode="r", encoding='utf-8') as f:
    for line in f:
        data = json.loads(line)
        lines.append(data)

with open("test.jsonl", mode="r", encoding='utf-8') as f:
    for line in f:
        data = json.loads(line)
        lines.append(data)

pretty = json.dumps(lines[2], indent=4)
print(pretty)
# pretty = json.loads(pretty)

# content = pretty["gold_contents"]
# for page in content:
#     print(page)

line = lines[2]
print(line)

{
    "question": "I'm planning a research project that will involve working with both radioactive tracers and arsenic-containing mineral samples in the same laboratory space. If there's an accident where both materials are spilled simultaneously - say a radioactive solution spills onto arsenic-containing rock samples - what are the priority procedures I need to follow, and how do the health monitoring and reporting requirements interact between these two types of hazardous materials?",
    "gold_answer": "This is a complex emergency scenario requiring careful coordination of procedures from both the radioactive spill and arsenic safety protocols. Here's the priority sequence and integrated requirements:\n\n**Immediate Emergency Response Priority:**\n1. **Medical treatment takes absolute priority** - if anyone is injured, activate UQ emergency procedures immediately (call 000 for ambulance, then Security 3365 3333), regardless of contamination concerns\n2. **Evacuate and control access

This is a complex emergency scenario requiring careful coordination of procedures from both the radioactive spill and arsenic safety protocols. Here's the priority sequence and integrated requirements:

**Immediate Emergency Response Priority:**
1. **Medical treatment takes absolute priority** - if anyone is injured, activate UQ emergency procedures immediately (call 000 for ambulance, then Security 3365 3333), regardless of contamination concerns
2. **Evacuate and control access** - remove all non-essential personnel from the area and establish a control point outside the laboratory
3. **Don appropriate PPE** before any cleanup attempts - this must protect against both radioactive and chemical contamination (laboratory coat, double gloves, safety glasses, overshoes, and respirator if dust/vapour present)

**Dual Notification Requirements:**
You must notify both specialist advisors immediately:
- **Radiation Safety Officer (RSO)** for the radioactive component - they will assess the incident and determine response requirements
- **HSW Division Occupational Hygiene Advisor** for the arsenic exposure risk assessment
- Provide both the Radiation Safety Data Sheet and arsenic Safety Data Sheet to respective specialists

**Containment and Monitoring:**
- Use only **wet decontamination methods** to prevent respirable dust (required for both materials)
- Wipe inward toward the center of the spill to prevent spread
- Conduct radiation monitoring using appropriate meters for the specific radionuclides
- The RSO will determine whether immediate decontamination or laboratory closure for decay is appropriate

**Health Monitoring Integration:**
This incident would trigger **both** monitoring protocols:
- **Radiation monitoring**: Personal dosimetry assessment, contamination surveys, and potential bioassay depending on the radionuclides involved
- **Arsenic health monitoring**: Urinary inorganic arsenic testing coordinated by the UQ Occupational Health Nurse Advisor, particularly if there was potential inhalation or skin contact during the incident

**Decontamination Hierarchy:**
1. Personnel decontamination (if contaminated) - remove clothing, wash with mild soap and water, then specialized decontaminants as needed
2. Area decontamination using appropriate agents for both contaminants
3. Equipment decontamination or disposal - some items may need to be treated as both radioactive and chemically hazardous waste

**Waste Management:**
The contaminated materials become **mixed waste** requiring specialized disposal through UQ Science Store, as they contain both radioactive and hazardous chemical components. Standard disposal routes for either material alone would not be appropriate.

**Reporting Requirements:**
You must report this incident in **UQSafe** with both the RSO and RPC listed as specialist advisors, and the report must address both radiation doses and potential arsenic exposure. The incident investigation must review both the Radiation Safety Protection Plan and arsenic handling procedures.

**Critical Integration Point:**
The RSO has authority to control the incident scene and may need to coordinate with the HSW Division to ensure both radiation safety and chemical safety protocols are properly implemented. Neither set of procedures should be applied in isolation when both hazards are present.